aii-pxelinux/src/main/pan/quattor/aii/pxelinux/config.pan

```diff
@@ -99,7 +99,7 @@ variable AII_NBP_ROOT ?= {
     if (LPXELINUX) {
         root = LPXELINUX_ROOT;
     } else {
-        toks =  matches(AII_OSINSTALL_OS_VERSION, '^(\w+?)[_\-](.*)');
+        toks =  matches(AII_OSINSTALL_OS_VERSION, '^([\w\.]+?)[_\-](.*)$');
```

Comment:
```suggestion
        toks = matches(AII_OSINSTALL_OS_VERSION, '^([\w\.]+?)[_\-](.*)$');
```

Reply:
Done

Reply:
Strange if was not found by `panlint`...